### PR TITLE
Add June blog post

### PR DESCRIPTION
**Pull request by gavD**
Blog post for June

WHAT U SAY

### CODE REVIEW

**Comment by gavD** [comment]
@rodnaph @craigmarvelley @jenkoian @handybitesize  sum1 merge it pls, Wolan has seen an' rated 10/10

File: blog/_posts/2015-05-10-venue-change-for-may-meetup.md [hunk 1]
```diff
@@ -45,3 +45,4 @@ chance to eat then don't worry we'll have a bunch of pizza available and some
 beverages to wash it all down.  Then sit back, enjoy the talks, and meet some
 new people.  See you there!
 
+- Rod, May 2015
```
Comment (jenkoian) [inline blog/_posts/2015-05-10-venue-change-for-may-meetup.md]: Link through to twitter on the names?

**Comment by jenkoian** [comment]
Great post :+1:

**Comment by handybitesize** [comment]
tl;dr;m

**Comment by rodnaph** [comment]
Great post, only suggestion I would have made is for you to quote yourself from your book. Fab!

**Pull request merged**